_config.yml

```diff
@@ -42,31 +33,98 @@ navigation:
   internal: true
 
 
-# Repo list
-# List repos that you would like to appear on the homepage here
-repos:
-- name: regulations-parser
-  description: Parse regulations
-  url: https://github.com/cfpb/regulations-parser
-- name: regulations-core
-  description: Regulations API
-  url: https://github.com/cfpb/regulations-core
-- name: regulations-site
-  description: Display the regulations
-  url: https://github.com/cfpb/regulations-site
-
-support_repos:
-- name: regulations-bootstrap
-  description: Bootstrap eRegulations
-  url: https://github.com/cfpb/regulations-bootstrap
+general_repos:
+  - name: regulations-parser
+    url: https://github.com/18F/regulations-parser
+    short: Parse regulations
+    long: >
+      The parser effectively converts print-centric documents from the Federal
+      Register, Government Printing Office, etc., pulls out semantic meaning
+      from them (e.g. paragraph structures, citations, definitions), and
+      writes those structures to a data store. The parser also contains tools
+      to help developers debug this process.
+  - name: regulations-core
+    url: https://github.com/18F/regulations-core
+    short: Regulations API
+    long: The store for the parsed regulatory data.
+  - name: regulations-site
+    url: https://github.com/18F/regulations-site
```

Comment:
Is it accurate to list the 18F repo as a "general" repo? If so, should we move ownership to this organization rather than keeping it under 18F?

Actually, I would say the same for any of the 18F repos listed as "general repos"